opencti-platform/opencti-graphql/src/modules/entitySetting/entitySetting.graphql

```diff
@@ -27,6 +27,15 @@ type DefaultValueAttribute {
   defaultValues: [DefaultValue!]!
 }
 
+type OverviewLayoutCustomizationParameter {
+  width: Int!
+}
+
+type OverviewLayoutCustomization {
+  key: String!
+  values: OverviewLayoutCustomizationParameter!
```

Comment:
This should be width: Int! directly

Reply:
No.
I want to keep the object to destructure it. And, explicitly name the constant. No magic number.

Reply:
I understand now.
Let me optimize the typing by changing the OverviewLayoutCustomizationParameter with the width property.